app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
Bạn đang dùng `gem devise` đúng không?
Xem lại `Controller filters and helpers` của devise để làm chỗ này nhé ;)